content/bots.md

```diff
@@ -3,9 +3,9 @@ layout: page
 title: 2factorauth Web Bots
 aliases: /bot
 ---
-In order to validate user contributions we use scripts or "bots".  
-These scripts only make requests to your website when someone tries to edit data about your site on 2fa.directory.   
-As a result you will most likely only receive a couple of requests each year. We would be very thankful if you didn't block these HTTP requests.
+To validate user contributions, we use scripts or "bots."  
+These scripts only make requests to your website when someone tries to edit data about your site on the 2FA Directory.   
```

Comment:
Should the site be referred to as "the 2FA Directory" or simply "2FA Directory"? I vote for the latter.

Reply:
You're the native speaker here so I trust your grammar over mine :wink: 
```suggestion
These scripts only make requests to your website when someone tries to edit data about your site on 2FA Directory.   
```